test/components/portal.spec.tsx

```diff
@@ -76,36 +79,37 @@ describe('<Portal />', function() {
             container
         );
 
-        await waitFor(() => expect(bodySelect('UPDATED_SPAN')).to.be.present());
+        await waitFor(() => expect(portal.content[0]).to.be.present());
```

Comment:
This test case doesn't seem to test what it's supposed to. We replace one span with another, and then simply check that the `portal.content` still has a first child - we don't actually check that the span changed.

We should check that there's only one child, that the child is a span, and that span has specific content.

Also, `portal.content` makes me think that there's something else might be present in the portal apart from its content. And by design we don't want anything else to present there. I think we should directly check `portal.root.children`;

Reply:
probably more convenient to reduce boilerplate property access and just go to portal.content